aws/sqs_event_manager/sqs_event_manager/message.py

```diff
@@ -56,5 +57,17 @@ def event_source_arn(self) -> str:
     def receipt_handle(self) -> str:
         return self.__get('receiptHandle')
 
+    def __jsonify_message(self, data) -> str:
```

Comment:
This seems more confusing then just using `json.loads` on the `Message` content. I tested and confirmed this with a production entitlement change event.